projects/storefrontapp-e2e-cypress/cypress/e2e/regression/cart/cart.e2e.cy.ts

```diff
@@ -20,8 +20,7 @@ describe('Cart', () => {
     });
 
     context('Registered user', () => {
-      before(() => {
-        cy.window().then((win) => win.sessionStorage.clear());
+      beforeEach(() => {
```

Comment:
if test fails, the following attempt will execute beforeEach hook which make the test independant and repeatable. Previous behavior, attempt 2 and 3 were stuck on blank window, as testIsolation triggers navigation to blank page before each test.